src/main/resources/l10n/JabRef_no.properties

```diff
@@ -968,53 +856,45 @@ no_database_generated=ingen_database_generert
 
 No_entries_found._Please_make_sure_you_are_using_the_correct_import_filter.=Ingen_enheter_funnet._Kontroller_at_du_bruker_riktig_importfilter.
 
+
 No_entries_found_for_the_search_string_'%0'=Fant_ingen_enheter_for_s\u00f8keteksten_'%0'
 
 No_entries_imported.=Ingen_enheter_importert.
 
-
-
 No_exceptions_have_occurred.=Ingen_feiltilstander_har_inntruffet.
-
 No_files_found.=Ingen_filer_funnet.
 
 No_GUI._Only_process_command_line_options.=Ingen_GUI._Bare_behandle_kommandolinjevalg.
 
 No_journal_names_could_be_abbreviated.=Ingen_journalnavn_kunne_forkortes.
 
 No_journal_names_could_be_unabbreviated.=Ingen_journalnavn_kunne_ekspanderes.
-
 No_PDF_linked=Ingen_PDF_linket
 
 No_references_found=Ingen_referanser_funnet
 
-
 No_URL_defined=Ingen_url_er_definert
-
 not=ikke
 
 not_found=ikke_funnet
 
-
 Note_that_you_must_specify_the_fully_qualified_class_name_for_the_look_and_feel,=Merk_at_du_m\u00e5_spesifisere_det_fullstendige_klassenavnet_for_utseendet,
 
 Nothing_to_redo=Ingenting_\u00e5_gjenta
 
 Nothing_to_undo=Ingenting_\u00e5_angre
 
-# The next is used like in "References found\: 1  Number of references to fetch?"
+#Thenextisusedlikein"Referencesfound\:1Numberofreferencestofetch?"
```

Comment:
The spaces are already missing in the current English localization.
Comments in languages other than English overridden by the English one.

https://github.com/JabRef/jabref/blob/master/src/main/resources/l10n/JabRef_en.properties#L520